src/main/java/seedu/address/model/person/Age.java

```diff
@@ -3,6 +3,10 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
+/**
```

Comment:
Good enough for 1.2, not necessary to be final since the future implementation will compare age to birthday/date